src/core/models/filter.js

```diff
@@ -23,6 +23,34 @@ export default class Filter {
     }
   }
 
+  /**
+   * Check whether this simple filter is equal to another simple filter.
+   * @param {Filter} filter
+   * @returns {boolean}
+   */
+  isEqualToSimpleFilter (filter) {
+    if (this.getFilterKey() !== filter.getFilterKey()) {
+      return false;
+    }
+    const fieldIdsToValues1 = this[this.getFilterKey()];
+    const fieldIdsToValues2 = filter[filter.getFilterKey()];
+    const fieldIds1 = Object.keys(fieldIdsToValues1).sort();
+    const fieldIds2 = Object.keys(fieldIdsToValues2).sort();
+    if (fieldIds1.length !== fieldIds2.length) {
```

Comment:
I think we should give these things better names. The 1 and 2 suffix aren't particularly descriptive. Are these objects maps of fieldId to value? I thought the filterKey itself was a fieldId.

Reply:
updated! not sure how much better my new names are, but this is kind of what lodash names their params in their readme